lib/hyacinth/adapters/preservation_adapter/fedora3/pid_helpers.rb

```diff
@@ -0,0 +1,12 @@
+module Hyacinth
+  module Adapters
+    module PreservationAdapter
+      module Fedora3::PidHelpers
+        def digital_object_pids(hyacinth_obj)
```

Comment:
```suggestion
        def digital_object_fedora_pid_uris(hyacinth_obj)
```
Suggesting this change because this method returns fedora object URIs rather than PIDs.